src/components/ListUniqueValues.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div class="list-unique-values">
+    <div>FILTER VALUES</div>
+    <input
+      class="list-unique-values__search-box"
+      placeholder="Search 50 items"
+      @input="search($event.target.value)"
+    />
+    <div class="list-unique-values__select-clear-all">
+      <span class="list-unique-values__select-all" @click="selectAll">Select all</span> &nbsp;
+      <span class="list-unique-values__clear-all" @click="clearAll">Clear all</span>
+    </div>
+    <div
+      class="list-unique-values__checkbox-container"
+      v-for="value in searchedValues"
+      :key="value.value"
+    >
+      <CheckboxWidget
+        :label="`${value.value} (${value.nbOcc})`"
+        :value="isChecked[value.value]"
+        @input="toggleCheck(value.value)"
+      />
+    </div>
+    <div class="list-unique-values__load-more" v-if="loaded">
+      <div>List maybe incomplete</div>
+      <div @click="loadMore" class="list-unique-values__load-more-button">Load more</div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+// This type will be imported from @/lib/dataset/helpers.ts
+type ColumnValueStat = {
+  value: any;
+  nbOcc: number;
+};
+
+import CheckboxWidget from './stepforms/widgets/Checkbox.vue';
+@Component({
+  name: 'filter-value',
+  components: { CheckboxWidget },
+})
+export default class ActionMenu extends Vue {
+  @Prop({
+    type: Array,
+    required: true,
+  })
+  values!: ColumnValueStat[];
+
+  @Prop({
+    type: Boolean,
+    required: true,
+  })
+  loaded!: boolean;
+
+  searchedValues: ColumnValueStat[] = this.values;
+
+  isChecked: Record<string, boolean> = Object.fromEntries(this.values.map(e => [e.value, true]));
+
+  selectAll() {
+    this.isChecked = Object.fromEntries(this.values.map(e => [e.value, true]));
+    this.$emit('input', Object.keys(this.isChecked));
+  }
```

Comment:
In Toucan Toco, we have a clever way to say "all values" or "all values except", which avoids sending all the actual values. Indeed, this could be huge if the column has only unique values.
Also, it's not the same to say: "I want every values except this one" and "I want this this and this values".

The Toucan model for this:
```ts
{
  type: 'include' | 'exclude',
  values: any[]
}
```